datasets/wiki_movies/README.md

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
Can you remove the files related to wiki_movies to only keep the ones for imppres ?

Reply:
What would be the best way to do this?

Reply:
delete them and commit the change should do the job

Reply:
Done!